test/javascript/tests/design_docs.js

```diff
@@ -10,7 +10,9 @@
 // License for the specific language governing permissions and limitations under
 // the License.
 
+couchTests.elixir = true;
```

Comment:
Is this something new?

Reply:
I just noticed that this is the correct way to tag the porting progress. I took a wrong reference for this.

Reply:
Oh! So we'll have to upgrade some :) the rest is perfect, just merge 👍

Reply:
Yes I think we should check the ported tests and tag it correctly